tests/Platforms/AbstractMySQLPlatformTestCase.php

```diff
@@ -808,6 +807,108 @@ protected function getQuotedAlterTableChangeColumnLengthSQL(): array
         ];
     }
 
+    public function testAlterTableChangeBlobColumnLength(): void
+    {
+        $fromTable = new Table('mytable');
+
+        $fromTable->addColumn('blob1', 'blob', ['length' => MySQLPlatform::LENGTH_LIMIT_TINYBLOB]);
+        $fromTable->addColumn('blob2', 'blob', ['length' => MySQLPlatform::LENGTH_LIMIT_BLOB]);
+        $fromTable->addColumn('blob3', 'blob', ['length' => MySQLPlatform::LENGTH_LIMIT_MEDIUMBLOB]);
+        $fromTable->addColumn('blob4', 'blob', ['length' => MySQLPlatform::LENGTH_LIMIT_MEDIUMBLOB * 2]);
+        $fromTable->addColumn('blob5', 'blob');
+        $fromTable->addColumn('blob6', 'blob', ['length' => MySQLPlatform::LENGTH_LIMIT_TINYBLOB]);
+        $fromTable->addColumn('blob7', 'blob', ['length' => MySQLPlatform::LENGTH_LIMIT_BLOB]);
+        $fromTable->addColumn('blob8', 'blob', ['length' => MySQLPlatform::LENGTH_LIMIT_MEDIUMBLOB]);
+        $fromTable->addColumn('blob9', 'blob', ['length' => MySQLPlatform::LENGTH_LIMIT_MEDIUMBLOB * 2]);
+        $fromTable->addColumn('blob10', 'blob');
```

Comment:
This will be really hard to maintain. As far as I understand, the test verifies that if a BLOB column length exceeds a certain threshold, the column type is changed. It could be done with a much simpler test and a data provider: i.e. declare one column of a given type, increase the length and assert that there is a diff. Repeat (via the data provider) for each size.

Reply:
I removed this tests now as they did not make much sense this way I think.